pom.xml

```diff
@@ -6,7 +6,7 @@
   <parent>
     <groupId>org.springframework.boot</groupId>
     <artifactId>spring-boot-starter-parent</artifactId>
-    <version>3.0.4</version>
+    <version>3.2.1</version>
```

Comment:
app-poc-tools and folio-spring-support have 3.2.0 version. Shall we use the same here?

Reply:
It's not necessary to use the same version, maven will handle it. And it make sense to use the latest version as it has many bug fixes